index.js

```diff
@@ -107,6 +112,30 @@ app.delete('/ideas', (req, res) => {
     res.status(200).send('Ideas successfully deleted.');
 });
 
+//DONATION METHODS
+
+app.get('/donations', (req, res) => {
+    res.status(200).json(ideasDonationSum(req.body.ideaId));
+});
+
+app.get('/donations/sumDifference', (req, res) => {
+    res.status(200).json(ideasSumDifference(req.body.ideaId));
+});
+
+app.post('/donations', (req, res) => {
+    try {
+        createDonationsWithDataValidation(
+            req.body.firstName,
+            req.body.sum,
+            req.body.userId,
+            req.body.ideaId
+        );
+    } catch (Error) {
+        res.status(404).send(Error.message);
+    }
+    res.status(200).send('Donation created successfully');
```

Comment:
I've noticed you're using 3 message formats:
1. Without `.`
2. With `.`
3. With `..`

Can you format all messages consistently please?

Reply:
Yes, I've formated. You are totally right.
And I've added some error messages in business logis\donations.js functions (where IdeaId was mentioned)